exercises/rotational-cipher/.meta/generator/rotational_cipher_case.rb

```diff
@@ -0,0 +1,9 @@
+require 'generator/exercise_case'
+
+class RotationalCipherCase < Generator::ExerciseCase
+
+  def workload
+    assert_equal { "RotationalCipher.rotate(\"#{text.to_s}\", #{shiftKey.to_s})" }
```

Comment:
Are the `.to_s` calls here necessary?

Using `%Q` here would help you by not needing to escape the `"`